src/main/java/picard/illumina/ExtractIlluminaBarcodes.java

```diff
@@ -174,6 +183,9 @@
             "the number available on the machine less NUM_PROCESSORS.")
     public int NUM_PROCESSORS = 1;
 
+    @Option(doc = "Use the new converter", optional = true)
```

Comment:
The rational here was that this new convert could (and likely will) support other sequencer types and replace the outdated internal machinery for data providers. That said this was left as an explicit argument because I would consider it experimental. I'm happy to change it to be all encompassing and do things under the covers.